Makefile

```diff
@@ -67,6 +67,7 @@ scwversion/version.go: .git
 
 $(BUILD_LIST): %_build: %_fmt %_iref
 	$(GOBUILD) -o $(NAME) ./$*
+	go tool vet -all=true $(PACKAGES)
```

Comment:
Actually this will run multiple time the same test, but it's not a real problem :)